src/content/en/_shared/feedback.html

```diff
@@ -3,13 +3,14 @@
   usage.
 {% endcomment %}
 {% framebox width="auto" height="auto" enable_widgets="true" %}
-{% setvar url %}https://github.com/google/webfundamentals/issues/new?title=[{{ category }} / {{ label }}] {% endsetvar %}
```

Comment:
I like the direction, but I think it'll cause "compile" errors. A feedback file like `/web/tools/chrome-devtools/javascript/_feedback/1.html` uses `feedbackURL` like so:

    {% setvar fail_response %}... please <a href="{{feedbackURL}}">file an issue</a>{% endsetvar %}

And then `1.html` includes `/web/_shared/feedback.html`. Since `feedbackURL` is used before it's declared, Django is going to throw an error.